rhel_sync.sh

```diff
@@ -30,7 +31,6 @@ declare -a repos=(
 )
 
 for repo in "${repos[@]}"; do
-  reposync -lnm -r $repo -e ${DEST_DIR}/cache -p ${DEST_DIR}
-  createrepo -v --workers 4 -c ${DEST_DIR}/cache --update -g comps.xml ${DEST_DIR}/${repo}/
+  reposync -lnm -r $repo -e ${MIRROR_TLD}/cache -p ${DEST_DIR}
```

Comment:
Does the cache file have to live under MIRROR_TLD?  If it doesn't can it be located on the outside of the repos dir?  That way we don't have to exclude it from any differential rsyncs?